### ISSUE TITLE
CCA column preview empty after Image Cropper 

### ISSUE THREAD

**Issue by stelfrich** [knime-ip/knip#494]
It seems that if _Adjust Dimensionality?_ is unchecked in an _Image Cropper_ node, the output tables of subsequent _Connected Component Analysis_ nodes do not show a preview of the labeling. Clicking on the cell shows the labeling as expected.

Example workflow: [CCAPreviewAfterImageCropper.zip](https://github.com/knime-ip/knip/files/1672594/CCAPreviewAfterImageCropper.zip)

**Comment by gab1one** [comment]
Thank you for reporting this, it seems to be a bug with our preview rendering.

**Comment by imagejan** [comment]
I noticed the same issue with *Adjust Dimensionality?* being checked and cropping an XYZ image with a defined Z range: after a *Global Thresholder* and a *Connected Component Analysis*, the thumbnails are missing in the output table.

**Comment by gab1one** [comment]
@stelfrich  @imagejan  This issue is related to the Image Cropper not renormalizing the axis minima. I created a metanode [ZeroMin metanode.zip](https://github.com/knime-ip/knip/files/1708255/ZeroMin.metanode.zip) which performs the needed ZeroMin operation. Just add this as a workaround to your workflows where the cropper behavior causes you issues. We might add this as a popper node to KNIP at a later point in time.

**Comment by imagejan** [comment]
I noticed that this can also problems when trying to use a legacy `ij.ImagePlus` as an input parameter in an ImageJ2 plugin.

I got:
```
ERROR ThreadPool                      An exception occurred while executing a runnable.
```
... when running my [`TrackMateSpotDetector`](https://github.com/fmi-faim/fmi-ij2-plugins/blob/88a8b74cc5b22927c719aea00d3ae21b704818c4/src/main/java/ch/fmi/TrackMateSpotDetector.java) on a cropped input image. (Strangely, it still worked when I was running the saved workflow, but threw these errors after I had reconfigured the node...)

Adding your *ZeroMin* metanode helped here, but I see no way of fixing this inside my own plugin, while still taking an `ImagePlus` input (and I'd like to avoid the need of a ZeroMin node in the final workflow, ideally).

**Comment by gab1one** [comment]
You could always run `Views.zeroMin(..)` on your input images? Something like:
```java
RAI myImp = Views.zeroMin(imp);
```

**Comment by imagejan** [comment]
@gab1one yes, of course, if I take an `ImgPlus` or `RAI`; but as I take an `ImagePlus` for using it with TrackMate, I wanted to avoid the additional wrapping, because that's what the framework is supposed to do correctly if I define an `@Parameter ImagePlus`, no?

**Comment by gab1one** [comment]
In most cases you will actually get an ImgPlus converted to a ImagePlus, at least in KNIME we always use IJ2 / Imglib2 data structures internally. We do not run `Views.zeroMin()` on our cropped image segments because then you lose the offset information, which you need if you want to recombine the segments at a later time (e.g. using compose image in the GroupBy node). 

Maybe we can add an improved conversion strategy in the future, until then I recommend you just request a ImgPlus / RAI as input, wrap it into an ImagePlus yourself after performing Views.zeroMin(..) on it. Then you have maximal control over it. I am sorry that there is currently no easier way for it.
